executor/show.go

```diff
@@ -238,6 +238,9 @@ func (v *visibleChecker) Leave(in ast.Node) (out ast.Node, ok bool) {
 
 func (e *ShowExec) fetchShowBind() error {
 	var bindRecords []*bindinfo.BindRecord
+	if !e.ctx.GetSessionVars().UsePlanBaselines {
+		e.ctx.GetSessionVars().StmtCtx.AppendWarning(errors.New("system variable tidb_use_plan_baselines is disabled, although some statuses of bindings are `using`, it doesn't affect query plans."))
```

Comment:
How about changing it to "System variable tidb_use_plan_baselines is disabled, although the statuses of some bindings are `using`, these bindings won't take effect in plan selecting".